tests/conftest.py

```
@@ -439,7 +439,86 @@ def _ensure_yarl() -> None:
+    selector.TextSelectorType = getattr(
+        selector,
+        "TextSelectorType",
+        type("TextSelectorType", (), {"PASSWORD": "password"}),
+    )
+    selector.TextSelectorConfig = getattr(
+        selector,
+        "TextSelectorConfig",
+        type(
+            "TextSelectorConfig",
+            (),
+            {"__init__": lambda self, type=None, **kwargs: setattr(self, "type", type)},
```

Ruff (0.16.1)

[error] Lambda argument `type` is shadowing a Python builtin

(A006)